etn-ccis/blui-react-workflows
# Include email argument in validateUserRegistrationRequest

Fixes [529](https://github.com/etn-ccis/blui-react-workflows/issues/529) .



#### Changes proposed in this Pull Request:

- send email details to validateUserRegistrationRequest
- Update changelog
-



#### Screenshots / Screen Recording (if applicable)

-



#### To Test:

- add a console to check email and code for `validateUserRegistrationRequest` action in  `login-workflow/example/src/actions/RegistrationUIActions.tsx`.
- go to self `http://localhost:3000/self-registration` 
- complete the steps and observe the console after verifyCodeScreen.



#### Any specific feedback you are looking for?

-
